swift/BUILDING.md

```diff
@@ -1,69 +1,54 @@
-# Ice for Swift Build Instructions
+# Build Ice for Swift
 
-This file describes how to build Ice for Swift from source and how to test the
-resulting build.
+This file describes how to build Ice for Swift from source and how to test the resulting build.
 
-- [Ice for Swift Build Instructions](#ice-for-swift-build-instructions)
-  - [Swift Build Requirements](#swift-build-requirements)
-    - [Operating Systems](#operating-systems)
-    - [Swift Version](#swift-version)
-  - [Building Ice for Swift](#building-ice-for-swift)
-    - [Building the C++ XCFrameworks](#building-the-c-xcframeworks)
-    - [Building with Make](#building-with-make)
-    - [Building with Xcode](#building-with-xcode)
-  - [Running the Swift Test Suite](#running-the-swift-test-suite)
-    - [macOS](#macos)
-    - [iOS](#ios)
+- [Building](#building)
+  - [Prerequisites](#prerequisites)
+  - [Building with Make](#building-with-make)
+  - [Building with Xcode](#building-with-xcode)
+- [Running the Swift Test Suite](#running-the-swift-test-suite)
+  - [macOS](#macos)
+  - [iOS](#ios)
 
-## Swift Build Requirements
+## Building
 
-### Operating Systems
+### Prerequisites
 
-Ice for Swift builds and runs on macOS and is supported on the platforms listed
-on the [supported platforms] page.
+1. Swift and Xcode\
+   You need Swift 5 or later with a recent version of Xcode for macOS.
 
-### Swift Version
+2. Ice C++ XCFrameworks\
+   Building Ice for Swift requires having first built Ice for C++ from the `cpp` directory. The Ice for C++ build
+   creates the following C++ `XCFrameworks`:
+    - `Ice.xcframework`
+    - `IceDiscovery.xcframework`
+    - `IceLocatorDiscovery.xcframework`
 
-Ice for Swift requires Swift 5 or later.
+   If you want to create binaries for iOS or the iOS simulator, build the C++ XCFrameworks with `PLATFORMS=all`:
 
-## Building Ice for Swift
+   ```shell
+   cd cpp
+   make PLATFORMS=all
+   ```
 
-Ice for Swift is built using the Swift Package Manager. The Swift Package Manager is included in the Swift toolchain,
-which is included with Xcode.
+### Building with make
 
-### Building the C++ XCFrameworks
-
-Building Ice for Swift requires having first built Ice for C++ from the `cpp` directory. This builds the required
-C++ `XCFrameworks`:
-
-- `Ice.xcframework`
-- `IceDiscovery.xcframework`
-- `IceLocatorDiscovery.xcframework`
-
-If targeting Swift for iOS (or iOS simulator), the XCFrameworks must also be built to include the iOS architectures:
-
-```shell
-cd cpp
-make PLATFORMS=all
-```
-
-### Building with Make
-
-To build Ice for Swift and its corresponding tests, run the following command from the `swift` directory:
+Run the following command from the `swift` directory:
 
 ```shell
 make
 ```
 
-The `PLATFORMS` argument can be used to include the [iOS test controller]:
+Add `PLATFORMS=all` to include the [iOS test controller] in your build:
 
 ```shell
 make PLATFORMS=all
 ```
```

Comment:
@externl Doesn't this require to configure the code signing, as you build for the iOS devices?

Reply:
In this case yes. I would just recommend doing the test controller build in Xcode or only do `PLATFORMS=macos iphonesimulator`